sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ExposureInContactEpiDataSteps.java

```diff
@@ -0,0 +1,278 @@
+              .isEqualToIgnoringCase(exposureDetailsInput.getHandlingSamples());
+          softly
+              .assertThat(exposureDetailsOutput.getTypeOfPlace())
+              .isEqualToIgnoringCase(exposureDetailsInput.getTypeOfPlace());
+          softly
+              .assertThat(exposureDetailsOutput.getContinent())
+              .isEqualToIgnoringCase(exposureDetailsInput.getContinent());
+          softly
+              .assertThat(exposureDetailsOutput.getSubcontinent())
+              .isEqualToIgnoringCase(exposureDetailsInput.getSubcontinent());
+          softly
+              .assertThat(exposureDetailsOutput.getCountry())
+              .isEqualToIgnoringCase(exposureDetailsInput.getCountry());
+          softly
+              .assertThat(exposureDetailsOutput.getExposureRegion())
+              .isEqualToIgnoringCase(exposureDetailsInput.getExposureRegion());
+          softly
+              .assertThat(exposureDetailsOutput.getDistrict())
+              .isEqualToIgnoringCase(exposureDetailsInput.getDistrict());
+          softly
+              .assertThat(exposureDetailsOutput.getCommunity())
+              .isEqualToIgnoringCase(exposureDetailsInput.getCommunity());
+          softly
+              .assertThat(exposureDetailsOutput.getStreet())
+              .isEqualToIgnoringCase(exposureDetailsInput.getStreet());
+          softly
+              .assertThat(exposureDetailsOutput.getHouseNumber())
+              .isEqualToIgnoringCase(exposureDetailsInput.getHouseNumber());
+          softly
+              .assertThat(exposureDetailsOutput.getAdditionalInformation())
+              .isEqualToIgnoringCase(exposureDetailsInput.getAdditionalInformation());
+          softly
+              .assertThat(exposureDetailsOutput.getPostalCode())
+              .isEqualToIgnoringCase(exposureDetailsInput.getPostalCode());
+          softly
+              .assertThat(exposureDetailsOutput.getCity())
+              .isEqualToIgnoringCase(exposureDetailsInput.getCity());
+          softly
+              .assertThat(exposureDetailsOutput.getAreaType())
+              .isEqualToIgnoringCase(exposureDetailsInput.getAreaType());
+          softly
+              .assertThat(exposureDetailsOutput.getCommunityContactPerson())
+              .isEqualToIgnoringCase(exposureDetailsInput.getCommunityContactPerson());
+          softly
+              .assertThat(exposureDetailsOutput.getGpsLatitude())
+              .isEqualToIgnoringCase(exposureDetailsInput.getGpsLatitude());
+          softly
+              .assertThat(exposureDetailsOutput.getGpsLongitude())
+              .isEqualToIgnoringCase(exposureDetailsInput.getGpsLongitude());
+          softly
+              .assertThat(exposureDetailsOutput.getGpsAccuracy())
+              .isEqualToIgnoringCase(exposureDetailsInput.getGpsAccuracy());
+        });
+  }
+
+  public void exposureIvestigationOnContact(ExposureInvestigation exposureInvestigationInput) {
```

Comment:
the name of the method doesn't reflect the acctions

Reply:
addressed